test/message/test_runner_output.js

```diff
@@ -328,3 +328,24 @@ test('subtest sync throw fails', async (t) => {
     throw new Error('thrown from subtest sync throw fails at second');
   });
 });
+
+test('timed out async test', { timeout: 5 }, async (t) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, 1000);
+  });
+});
+
+test('timed out callback test', { timeout: 5 }, (t, done) => {
+  setTimeout(done, 1000);
+});
+
+
+test('large timeout async test is ok', { timeout: Number.MAX_VALUE }, async (t) => {
```

Comment:
This seems strange, I'd expect this to run forever, but doesn't this behave exactly like the default 30s timeout because `Number.MAX_VALUE` is not an unsigned 32-bit integer?

Reply:
if the returned promise is fulfilled, why would it run forever?

Reply:
Sorry, I mean _virtually_ forever. What if it is fulfilled after two hours? Doesn't this test use the default timeout of 30s?